src/main/java/nextstep/courses/domain/course/CourseDetailInfo.java

```diff
@@ -0,0 +1,38 @@
+package nextstep.courses.domain.course;
+
+import java.util.Objects;
+
+public class CourseDetailInfo {
+
+  public static final String COURSE_TITLE_IS_INCORRECT = "과정 명이 올바르지 않습니다. 다시 확인해주세요. input: %s";
+  public static final String NEGATIVE_NUMBER_OR_ZERO_IS_NOT_ALLOWED = "유저 id는 0이나 음수일 수 없습니다. input: %s";
+
+  private final Long creatorId;
+  private final String titleName;
+
+  public CourseDetailInfo(Long creatorId, String titleName) {
+    validate(creatorId, titleName);
+    this.creatorId = creatorId;
+    this.titleName = titleName;
+  }
+
+  public Long getCreatorId() {
+    return creatorId;
+  }
+
+  public String getTitleName() {
+    return titleName;
+  }
+
+  private boolean validate(Long creatorId, String title) {
+    if (creatorId <= 0) {
+      throw new IllegalArgumentException(String.format(NEGATIVE_NUMBER_OR_ZERO_IS_NOT_ALLOWED, creatorId));
+    }
+
+    if (Objects.isNull(title) || title.isBlank()) {
```

Comment:
```suggestion
    if (title == null) || title.isBlank()) {
```

<img width="670" alt="image" src="https://github.com/next-step/java-lms/assets/36617783/1613cce0-4cd9-4c0d-a396-c2795ba71835">

결과적으로 같은 코드인데요
javadoc을 읽어보면 메소드 레퍼런스 표현을 위해 사용하는 메소드라고 하네요 (Objects::isNull)

직접 null 체크하는 부분이 마음에 들지 않으셨나보네요 ㅎㅎ